src/main/java/edu/cornell/kfs/sys/service/impl/CUMarshalServiceImpl.java

```diff
@@ -121,6 +126,12 @@ public <T> T unmarshalString(String xmlString, Class<T> clazz) throws JAXBExcept
         StringReader reader = new StringReader(xmlString);
         return (T) unmarshaller.unmarshal(reader);
     }
+    
+    @Override
+    public <T> T unmarshalStringIgnoreDtd(String xmlString, Class<T> clazz) throws JAXBException, XMLStreamException, IOException {
+        InputStream axmlStream = IOUtils.toInputStream(xmlString);
```

Comment:
Minor typo in variable name.

Also, is there a variant of `toInputStream()` that allows you to set the encoding?